src/components/NavBar/NavbarDesktop.jsx

```diff
@@ -17,7 +17,7 @@ import { useTheme } from '@mui/material/styles';
 // Component Imports
 import NavbarLinks from './NavbarLinks';
 import NavMenu from './NavMenu';
-import { SignedInUserContext } from '../../contexts';
+import { MessageContext, SignedInUserContext } from '../../contexts';
```

Comment:
```suggestion
import { MessageContext, SignedInUserContext } from '@contexts';
```
Not related to your changes, but will make things cleaner.